doc/build/dts/bindings-syntax.rst

```diff
@@ -73,8 +73,18 @@ These keys are explained in the following sections.
 Title
 *****
 
-Short description of the bound device, typically the hardware model.
-(It's optional.)
+Short description of the bound device, typically the hardware model. It should
+typically be on the format "Vendor Family Model". If acronyms are used, they
+should be spelled out in parentheses. The naming should stay as close to the
+vendor datasheet as possible.
+
+Titles should not exceed 100 characters. The description field should be used
+for longer descriptions. The words "binding" or "schema" should not be used
+in the title, everything is a binding.
```

Comment:
suggesting to also add "driver" as a discouraged term. In fact, we should also discourage the usage of "driver" in descriptions as well. (see https://docs.kernel.org/devicetree/bindings/writing-bindings.html#:~:text=DON%E2%80%99T%20refer%20to%20Linux%20or%20%E2%80%9Cdevice%20driver%E2%80%9D%20in%20bindings.%20Bindings%20should%20be%20based%20on%20what%20the%20hardware%20has%2C%20not%20what%20an%20OS%20and%20driver%20currently%20support.)

```suggestion
for longer descriptions. The words "binding", "schema", or "driver" should not be used
in the title, everything is a binding.
```

Reply:
Done!